mmc-google-analytics/src/main/java/info/novatec/metricscollector/google/aqeblog/AqeBlogReportRequestBuilder.java

```diff
@@ -24,26 +27,23 @@
     private GaConfigProperties gaConfigProperties;
```

Comment:
I know, it's not part of this PR. Therefore its just a tip for next implementations..
The type "GaConfigProperties" sounds a bit clumsy. You should avoid abbreviations. I have to know what "ga" means. In this case its obviously google analytics. In other cases its not that clear. Its better to write it out. "GoogleAnalyticsConfigProperties". Since the context is clear, in this case "ConfigProperties" would suffice. To my mind its ok to write "GoogleAnalyticsConfigProperties properties = ..."